meta-luneos/recipes-webos/com.webos.service.mediarecorder/com.webos.service.mediarecorder/0001-media_recorder_manager.cpp-Fix-format-specifier.patch

```diff
@@ -0,0 +1,45 @@
+From 4da309dc361403aae646a764a7c80850240fc2c5 Mon Sep 17 00:00:00 2001
+From: Herrie <Github.com@herrie.org>
+Date: Thu, 28 Sep 2023 08:17:14 +0200
+Subject: [PATCH] media_recorder_manager.cpp: Fix format specifier
+
+Using %ld here causes issues on armv7 (hammerhead) for example, due to size_t being unsigned integer type.
+
+Using %zu here will allow it to compile on armv7, aach64 and x86/64.
+
+Fixes:
+
+/media/herrie/LuneOS/kirkstone/webos-ports/tmp-glibc/work/cortexa8t2hf-neon-webos-linux-gnueabi/com.webos.service.mediarecorder/1.0.0-1+gitAUTOINC+ab3652ecfc-r0/git/include/log.h:46:58: error: format '%ld' expects argument of type 'long int', but argument 12 has type 'std::map<int, std::unique_ptr<MediaRecorder> >::size_type' {aka 'unsigned int'} [-Werror=format=]
+   46 |     PmLogInfo(getRecorderLunaPmLogContext(), LOG_TAG, 0, "[%d] %s():%d " FORMAT__, gettid(),       \
+/media/herrie/LuneOS/kirkstone/webos-ports/tmp-glibc/work/cortexa8t2hf-neon-webos-linux-gnueabi/com.webos.service.mediarecorder/1.0.0-1+gitAUTOINC+ab3652ecfc-r0/git/src/media_recorder_manager.cpp:502:5: note: in expansion of macro 'PLOGI'
+  502 |     PLOGI("recorders size %ld", recorders.size());
+      |     ^~~~~
+/media/herrie/LuneOS/kirkstone/webos-ports/tmp-glibc/work/cortexa8t2hf-neon-webos-linux-gnueabi/com.webos.service.mediarecorder/1.0.0-1+gitAUTOINC+ab3652ecfc-r0/git/src/media_recorder_manager.cpp:502:29: note: format string is defined here
+  502 |     PLOGI("recorders size %ld", recorders.size());
+      |                           ~~^
+      |                             |
+      |                             long int
+      |                           %d
+cc1plus: all warnings being treated as errors
+
+Signed-off-by: Herman van Hazendonk <github.com@herrie.org>
+---
```

Comment:
Add Upstream-Status

and you can way it's backport as this is already fixed in submission 2 (whenever that gets public with "ose: com.webos.service.mediarecorder=2").

Reply:
s/way/say/g